python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -74,6 +74,11 @@ class SpanAttributes:
     Total number of tokens, including both prompt and completion.
     """
 
+    LLM_TOOLS = "llm.tools"
```

Comment:
@RogerHYang is the change to this file intentional? It might confuse release please.